AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -25,8 +23,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign) BOOL isMutable;
 
-@property (nonatomic, strong, nullable) ASTraitCollection *traitCollection;
```

Comment:
@rcancro  We could certainly support an accessor that constructs and returns the nicer object (and then doesn't hold a reference to it).  This seems not required right now though, just a thought if you think it would be useful.